frontend/src/pages/projects/projectSharing/ProjectSharing.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import {
+  EmptyState,
+  EmptyStateBody,
+  EmptyStateIcon,
+  EmptyStateVariant,
+  PageSection,
+  Spinner,
+  Stack,
+  StackItem,
+  Title,
+} from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+import { ProjectDetailsContext } from '~/pages/projects/ProjectDetailsContext';
+import { GroupKind, UserKind } from '~/k8sTypes';
+import { ProjectSharingRBType } from './types';
+import ProjectSharingTableSection from './ProjectSharingTableSection';
+
+const ProjectSharing: React.FC = () => {
+  const {
+    projectSharingRB: { data: roleBindings, loaded, error: loadError, refresh: refreshRB },
+    users: { data: users },
+    groups: { data: groups },
+    currentProject,
+  } = React.useContext(ProjectDetailsContext);
+
+  if (loadError) {
+    return (
+      <PageSection isFilled aria-label="project-sharing-error-section" variant="light">
+        <EmptyState variant={EmptyStateVariant.large} data-id="error-empty-state">
+          <EmptyStateIcon icon={ExclamationCircleIcon} />
+          <Title headingLevel="h4" size="lg">
```

Comment:
I might think this is a a11y issue, not sure if we need h2 (same in other titles)

Reply:
Heading level is an axe issue I think because it's a semantic issue with the DOM. PF disables the sizing of h# tags... so use what you need to use based on your page hierarchy... h1 is probably always consumed by containers outside of you. h2 is likely what you want most of the time.